source/faq.rst

```diff
@@ -73,7 +73,8 @@ Push to cachix:
 
 .. code:: shell-session
 
-    $ nix-store -qR --include-outputs $(nix-instantiate shell.nix) | cachix push mycache
+    $ nix-store --query --references $(nix-instantiate shell.nix) | xargs nix-store --realise | xargs nix-store --query --requisites | cachix push mycache
```

Comment:
This also removes the need for previous command so that can be removed.

Reply:
I kind of like that echo statement; it's a really simple validation that the shell isn't borked